feat: Utilise create-discord-bot (#10013)
discordjs/discord.js: apps/guide/src/content/02-getting-started/01-starting-out.mdx

	higher, then you're good to go!
</Alert>

### Windows

- Download from the [Node.js website](https://nodejs.org).
- Use [fnm](https://github.com/Schniz/fnm).
- Use [Volta](https://volta.sh).

### macOS

- Download from the [Node.js website](https://nodejs.org/).
- Use [fnm](https://github.com/Schniz/fnm).
- Use [Homebrew](https://formulae.brew.sh/formula/node).
- Use [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating).
- Use [Volta](https://volta.sh).

### Linux

- Visit [this page](https://nodejs.org/en/download/package-manager) to determine how you should install Node.js.
- Use [fnm](https://github.com/Schniz/fnm).
- Use [nvm](https://github.com/nvm-sh/nvm).
- Use [Volta](https://volta.sh).

After installing Node.js, you'll be able to create a new application from your desired package manager. If you're starting out fresh, installing Node.js will also install npm, a package manager for Node.js.

<CH.Code lineNumbers={false} showCopyButton={true}>

```sh npm
npm create discord-bot
```

```sh yarn
yarn create discord-bot
```

```sh pnpm
pnpm create discord-bot
```

```sh bun
bun create discord-bot
```

</CH.Code>

> You'll be asked the directory to create the application in, as well as whether TypeScript should be used. Dependencies will be automatically installed for you. After this, you've just got your startup Discord bot template _nearly_ ready!

Reviewer: ```suggestion
You'll be asked the directory to create the application in, as well as whether TypeScript should be used. Dependencies will automatically be installed for you. After this, you've just got your startup Discord bot template _nearly_ ready!
```

> Next, we'll need to create an application so we are authenticated use Discord's API. This will be covered in the next section.

Reviewer: ```suggestion
In the next section, we will explain how to create an application to interact with Discord's API
```
